test/fixtures/movie_people.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Fixtures were generated by default, are they going to be used ? I see you got faker for generation task. If they're not required you can remove them